src/tests/adding.md

```diff
@@ -148,6 +148,9 @@ exhaustive. Header commands can generally be found by browsing the
 `TestProps` structure found in [`header.rs`] from the compiletest
 source.
 
+* `run-rustfix` for UI tests, indicates that the test produces
+  structured suggestions, which are then applied and the final
```

Comment:
Does it get compiled again? I think the output of rustfix is just compared against the .fixed file.

Reply:
Yes, [it gets compiled again](https://github.com/rust-lang/rust/blob/b4463d788bfd30b622a87a0e6f8e9271b9102e50/src/tools/compiletest/src/runtest.rs#L2634-L2649)

Reply:
Ah, then @oli-obk, could you just add that the expected fixed code should be in .fixed? Then r=me